doc/TUTORIAL.md

```diff
@@ -3,48 +3,40 @@ Tutorial on how to add a badge for a service
 
 This tutorial should help you add a service to shields.io in form of a badge.
 You will need to learn to use JavaScript, git and Github.
-Please [improve the tutorial][edit] while you read it.
+Please [improve the tutorial](https://github.com/badges/shields/edit/master/doc/TUTORIAL.md) while you read it.
 
 (1) Reading
 -----------
 
-You should read the following:
+You should read [CONTRIBUTING.md](../CONTRIBUTING.md)
 
-- [CONTRIBUTING.md](../CONTRIBUTING.md)
-
-You can read
-
-- previous successful pull-requests to see how other people implemented their badges.
-  Usually they start with "[add][add-pr]".
-- later [pull-requests tagged with `new-badge`][new-badge].
-- [implementations and their commits][blame] in the [server.js][server] file.
-- [merged pull-requests][mergedpr].
+You can also read previous
+[merged pull-requests with the 'service-badge' label](https://github.com/badges/shields/pulls?utf8=%E2%9C%93&q=is%3Apr+label%3Aservice-badge+is%3Amerged)
+to see how other people implemented their badges.
 
 (2) Setup
 ---------
 
 I suppose you have [git](https://git-scm.com/) installed.
```

Comment:
This is the last remaining bit using first person, could we change it?

Reply:
Tbh, there is probably a bit of a mashup of styles in the document at this point. The previous tutorial was mostly written in a more "instructive" tone e.g: "you should do this to your code". With the stuff I've added/edited I've mostly gone with a more "inclusive" tone e.g: "we will do this to our code". This is a quite a subtle language style point but it is probably useful to standardise one way or the other. Do you prefer the we/our style?

Reply:
I don't mind whether it's "you" or "we", but I think we should avoid "I" as it sounds as if one authority is dictating the tutorial.